consumptions/views.py

```diff
@@ -104,6 +103,6 @@ def upload_product_csv(request):
                     category = Product_Category.objects.get(code=category)
                     # if category not in Product_Category.objects.all():
                     if fields[0] not in db_codes:
-                        p = Product(code=code, name=name, category=category).save()
```

Comment:
I strongly recommend you integrate flake8 into your editor so you catch these things as early as possible thus, they don't get committed and we avoid the cost of having to remove them later.